src/Psl/DateTime/Timezone.php

```diff
@@ -487,60 +485,94 @@ enum Timezone : string
     case PacificWallis = 'Pacific/Wallis';
 
     /**
-     * Returns the default system time zone as a {@see Timezone} enum instance.
+     * Calculates the total time zone offset for a given {@see TemporalInterface} instance.
+     *
+     * This total offset includes both the raw timezone offset and any daylight saving time (DST) adjustments applicable at the temporal instance's time.
      *
-     * This method attempts to determine the current default time zone set in the system and return
-     * its corresponding {@see Timezone} enum instance. If the system's default time zone does not match any
-     * of the enum cases, UTC is returned as the default.
+     * @param bool $local Indicates whether the temporal object's time should be treated as local time (`true`) or as UTC time (`false`).
      *
-     * @return Timezone The system's default time zone as a {@see Timezone} enum instance or UTC if not found.
+     * @return Duration The total offset from UTC as a Duration instance, including any DST adjustments.
+     *
+     * @mutation-free
      */
-    public static function default(): Timezone
```

Comment:
no more "default" (/system) timezone?
It did make sense to me to have it. Otherwise you need to explicitly set it everywhere.

Reply:
Yes, that is the idea.

I'm still not sure about it however, we either have default timezone/locale, or not.

If yes, that means we will be accessing global state, e.g:

```php
$a = DateTime\DateTime::now()->getTimezone();

something_else();

$b = DateTime\DateTime::now()->getTimezone();
```

there will be no garuntee that both `$a` and `$b` are the same.

we could store the value interanlly ourselves to ensure future calls return the same value, but then, how do you change the default value at runtime?

Note: `Psl\Locale\Locale` also does not have 'default()` method, so if we are adding one here, we should add one for locale as well.

Reply:
I have added `Timezone::default()` and `Locale::default()`, neither are pure.